tests/auth.login.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { test, expect } from '@playwright/test';
+import path from 'path';
+
+const authFile = path.join(__dirname, '../playwright/.auth/user.json');
+
+test('Verify login', async ({ page }) => {
+  await page.goto('https://practicesoftwaretesting.com/');
```

Comment:
please don't hardcode URLs in the tests, as they might change. 
you already have baseUrl, so you can simplify it like that:
```suggestion
  await page.goto('/');
```